apps/desktop-wallet/src/components/AddressRow.tsx

```diff
@@ -16,33 +16,32 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
+import { AddressHash } from '@alephium/shared'
 import styled from 'styled-components'
 
 import AddressBadge from '@/components/AddressBadge'
 import AddressColorIndicator from '@/components/AddressColorIndicator'
 import { TableRow } from '@/components/Table'
-import { Address } from '@/types/addresses'
 
 interface AddressRowProps {
-  address: Address
-  disableAddressCopy?: boolean
-  onClick?: (address: Address) => void
+  addressHash: AddressHash
+  onClick?: (addressHash: AddressHash) => void
   className?: string
 }
 
-const AddressRow: FC<AddressRowProps> = ({ address, disableAddressCopy, onClick, children, className }) => (
+const AddressRow: FC<AddressRowProps> = ({ addressHash, onClick, children, className }) => (
```

Comment:
nit: I know it's not part of this PR, but should we remove `FC` for consistency sake?

Reply:
We have an old issue for it: https://github.com/alephium/alephium-frontend/issues/75

Reply:
Yep! I thought we said that we can make the changes as we go, when we stumble upon it, but I may be mistaken. Anyway, just a nit!